language-server/dm/document.ml

```diff
@@ -189,6 +190,9 @@ let record_outline document id (ast : Synterp.vernac_control_entry) classif (out
         match pure with
         | Vernacexpr.VernacStartTheoremProof (kind, _) -> Some (TheoremKind kind)
         | Vernacexpr.VernacDefinition ((_, def), _, _) -> Some (DefinitionType def)
+        | Vernacexpr.VernacInductive (kind, _) -> Some (InductiveType kind)
+        | Vernacexpr.VernacFixpoint (_, _) -> Some (DefinitionType Decls.Fixpoint)
+        | Vernacexpr.VernacCoFixpoint (_, _) -> Some (DefinitionType Decls.CoFixpoint)
```

Comment:
Are you sure these are really needed?

Reply:
I wasn't sure, I added it to cover all my bases. I tested it now, and from what I can tell the fix works without this change, so I removed this part of the diff.

Reply:
Thanks @SkySkimmer

Reply:
you don't need the Inductive case here since there's no such thing as an interactive Inductive